src/main/resources/openapi/headers/okapi-tenant.yaml

```diff
@@ -0,0 +1,6 @@
+in: header
```

Comment:
This overwrites the file from https://github.com/folio-org/folio-vertx-lib/tree/master/core/src/main/resources/openapi/headers
If the file is the same there is no need to overwrite it. If it is different a different file name should be used, for example okapi-tenant-required.yaml.
The same for okapi-token.yaml and okapi-url.yaml.

Reply:
Ok thanks. I removed the yaml file for token (since isn't required and the underlying folio-vertx-yaml handles that case). I renamed the two files (url and tenant) to be `-required` since they change that underlying behavior.